manifest.konnector

```diff
@@ -27,13 +27,16 @@
   ],
   "screenshots": [],
   "permissions": {
+    "shortcuts": {
+      "type": "io.cozy.files.shortcuts",
+      "verbs": [
+        "ALL"
+      ]
+    },
```

Comment:
`io.cozy.files.shortcuts` is not a real doctype. By real doctype I mean `database`. So you don't need to ask permissions about it, you have the rights on the `io.cozy.files` this is enough :).

https://github.com/cozy/cozy-doctypes/tree/master/docs : you'll not find `io.cozy.files.shortcuts` here. So since you'll not find it there, you can assume that it's an internal cozy-stack's doctype.

Reply:
Changes on the way !

Reply:
This commit should be removed.